support-payment-api/.prout.json

```diff
@@ -0,0 +1,5 @@
+{
+  "checkpoints": {
+    "PROD": { "url": "https://payment.guardianapis.com/healthcheck", "overdue": "15M" }
```

Comment:
@Jpsi this will interest you - there was a small unanticipated issue caused by moving this file into the mono-repo - there are now two Prout checkpoints called `PROD` in the https://github.com/guardian/support-frontend monorepo - the one newly moved in above, and the one already present here:

https://github.com/guardian/support-frontend/blob/291a4caf73108bdc06442e5ef7bb02a48fd6c25e/.prout.json#L2-L4

...so Prout doesn't accept there being two checkpoints with the same name in the same GitHub repo - which kind of makes sense because it uses those checkpoint names in the GitHub labels it generates, and it's ambiguous if there are two different `PROD`s.

Unfortunately Prout just throws an exception for this particular error if you attempt to look at the diagnostic for this:

https://prout-bot.herokuapp.com/view/guardian/support-frontend

![image](https://user-images.githubusercontent.com/52038/91973442-30147f80-ed14-11ea-885c-698999da90bf.png)

The exception -not visible to the user, but [visible in eg Sentry](https://sentry.io/organizations/the-guardian/issues/1872374324/?project=49913) - is the only helpful message saying what's gone wrong:

![image](https://user-images.githubusercontent.com/52038/91973465-3a367e00-ed14-11ea-9b69-7d4c0456bbc0.png)

Prout could be improved to report this problem better - but in the meantime, if the checkpoint here is renamed from `PROD` to something like `PROD-payment-api` then the error will resolve itself.

Reply:
@rtyley thank you, that makes sense! (Also, thank you for showing how to diagnose it, I wasn't aware of Sentry and it makes it very clear 👍)